app/scripts/lib/createRPCMethodTrackingMiddleware.js

```diff
@@ -60,6 +60,8 @@ const ETH_METHOD_TO_APPROVAL_TYPE = {
   [EthMethod.PersonalSign]: ApprovalType.PersonalSign,
   [EthMethod.Sign]: ApprovalType.Sign,
   [EthMethod.SignTransaction]: ApprovalType.SignTransaction,
+  // This method does not seem to be available in the EthMethod but supported by the test dapp
+  eth_signTypedData: ApprovalType.EthSignTypedData,
```

Comment:
I know it's not introduced your PR but can we use `MESSAGE_TYPE` from `shared/constants/app.ts` and remove `EthMethod` from this file?
In order to do that we must add `eth_signTypedData_v1` into `MESSAGE_TYPE` as well which is totally fine.

Reply:
Also the MESSAGE_TYPE misses eth_signTransaction and eth_signTypedData_v1

Reply:
done @OGPoyraz

Reply:
Nice! I was confused with what this type in this middleware represented.

I assumed ETH_METHODS since these are coming from the JSON_RPC request and it seems I assumed incorrectly. Thanks for the catch and the fix! cc: @pnarayanaswamy @OGPoyraz 

--- 

side note, though related: I have in my notes to understand more the differences and use-cases for our various transaction-related types. If anyone has more intel into any specifics of these, please share 

Eth Methods - EthMethod
Signatures - SIGNING_METHODS 
Transactions - TransactionType

Approvals - ApprovalType
Confirmations - CONFIRMATION_METHODS
Message - MESSAGE_TYPE